Gemfile

```diff
@@ -94,9 +94,11 @@ gem 'baby_squeel', '~> 1.4.3'
 gem 'ransack', '2.4.1' # we can remove line when stop using ruby 2.4
 gem 'browser', '~> 5.0.0' # we can update to lts when we stop using ruby 2.4
 gem 'diff-lcs', '~> 1.2'
+gem 'fastimage'
 gem 'hiredis', '~> 0.6.3'
 gem 'httpclient', github: 'mikz/httpclient', branch: 'ssl-env-cert'
 gem 'json-schema', git: 'https://github.com/3scale/json-schema.git'
+gem 'local-fastimage_resize', require: 'fastimage/resize'
```

Comment:
We're missing the minimum gem versions here.

Reply:
We don't really have a minimum as long as it works. But it is also in `Gemfile.lock`. I don't see how we can go below that.

Reply:
Done. Actually I noticed that we do not need `fastimage`. Because it uses another gem - `local-fastimage`.